### ISSUE TITLE
Digits lost during conversion from Int128

### ISSUE THREAD

**Issue by saschatimme** [JuliaMath/DoubleFloats.jl#102]
I just found this
```julia
julia> Double64(3503844571120554610950) - big(3503844571120554610950)
-135430.0
```

whereas

```julia
julia> Double64(big(3503844571120554610950)) - big(3503844571120554610950)
0.0
```

**Comment by JeffreySarnoff** [comment]
good catch

**Comment by JeffreySarnoff** [comment]
fixed in v1.1.7 (merge pending)

**Comment by JeffreySarnoff** [comment]
works -- thanks again
